client/src/components/MainContents/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react';
+import styled from '@emotion/styled';
+
+interface IMainContentsProps {
+  children: ReactNode | ReactNode[];
+}
+
+const MainContents = (props: IMainContentsProps): JSX.Element => {
+  return <Wrapper>{props.children}</Wrapper>;
+};
+
+const Wrapper = styled.main`
+  margin: 40px auto;
+  width: 1200px;
+`;
```

Comment:
엇 이런 형태라면 Link에 스타일을 추가할 필요 없이 여기서 자식 태그로 a 태그의 스타일링을 추가해줘도 될 것 같네요!
```suggestion
const Wrapper = styled.main`
  margin: 40px auto;
  width: 1200px;
  
  a {
    text-decoration: none;
    color: inherit;
  }
`;
```

Reply:
헤더에서 사용되는 link는 적용이 안될 것 같아요! 😢
채은님 말씀처럼 reset.css에 적용해줘도 좋은 것 같습니당.

Reply:
아 그렇네요! 좋습니당 :D